src/99-appendices/11-qmri.md

```diff
@@ -0,0 +1,547 @@
+
+```
+ ds-example/
+ └── derivatives/
+     └── qMRLab/
+         ├── dataset_description.json
+         └── sub-01/
+             └── anat/
+                 ├── sub-01_T1map.nii.gz
+                 ├── sub-01_T1map.json
+                 ├── sub-01_M0map.nii.gz
+                 └── sub-01_M0map.json
+```
+
+
+```
+dataset_description.json: 
+{
+  "Name": "qMRLab Outputs",
+  "BIDSVersion": "1.5.0",
+  "DatasetType": "derivative",
+  "GeneratedBy": [
+    {
+      "Name": "qMRLab",
+      "Version": "2.4.1",
+      "Container": {
+        "Type": "docker",
+        "Tag": "qmrlab/minimal:2.4.1"
+        }
+    },
+    {
+      "Name": "Manual",
+      "Description": "Generated example T1map outputs"
+    }
+  ],
+  "SourceDatasets": [
+    {
+      "DOI": "DOI 10.17605/OSF.IO/K4BS5",
+      "URL": "https://osf.io/k4bs5/",
+      "Version": "1"
+    }
+  ]
+}
+```
+
+In addition to the metadata fields provided in the `dataset_description.json`, qMRI maps
+are RECOMMENDED to be accompanied by sidecar JSON files that contain further information
+about the quantified maps. Although this may not be the generic case for common derivative
+outputs, a proper interpretation of qMRI maps may critically depend on some metadata
+fields. For example, without the information of `MagneticFieldStrength`, white-matter T1
+values in a `T1map` become elusive.
+
+* All the acquisition parameters that are constant across the files in a file collection
+are RECOMMENDED to be added to the sidecar json of the qMRI maps.
+* Relevant acquisition parameters that vary across files in a qMRI file collection are
+RECOMMENDED to be added to the sidecar json of the qMRI map **`in array form`**.
+* The JSON file accompanying a qMRI map which is obtained by using open-source software
+is RECOMMENDED to include additional metadata fields listed in the following table:
+
+| **Field name**                  | **Definition**                                                                                                               |
+| ------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `BasedOn`                       | List of files in a file collection to generate the map. Fieldmaps are also listed, if involved in the processing.            |
+| `EstimationReference`           | Reference to the study/studies on which the implementation is based.                                                         |
+| `EstimationAlgorithm`           | Type of algoritm used to perform fitting (e.g. linear, non-linear, LM etc.)                                                  |
+| `Units`                         | Units of the maps, in accordance with the BIDS specification.                                                                |
+
+Example:
+
+```Text
+sub-01_T1map.nii.gz
+sub-01_T1map.json
+```
+
+```
+sub-01_T1map.json
+{
```

Comment:
same comment as above (code syntax, and pulling description out of code block)

Reply:
https://github.com/bids-standard/bids-specification/pull/690/commits/274e1f82eade7d6d74d59bb6b5d9ebcf0b4c34a4